src/Error/GraphQLException.php

```diff
@@ -96,6 +103,7 @@ public function __construct(
         $this->resolveLocations($positions, $source);
 
         $this->path              = $path;
+        $this->extensions        = $extensions;
```

Comment:
@crisu83 What do we use extensions for?

Reply:
It's in the spec, you can add error codes, or whatever else you like there.